### DISCUSSION TITLE
🐛 Bug: how to find my database in my computer? I want back up my table and attachments,so that I can deploy on a new device using docker?

### DISCUSSION THREAD

**Discussion by doriszhang2020** [nocodb/nocodb#8359] [General]
### Please confirm if bug report does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

I cant find my database in my computer,I dont know where it is. I want back up my table and attachments,so that I can deploy on a new device using docker.Could u tell me how to do it,thanks.
![Quicker_20240428_195808](https://github.com/nocodb/nocodb/assets/104901283/b6558518-7962-4e95-bb9a-3d4a4d611a3a)

### Desired Behavior

-

### Project Details

-

### Attachments

-

**Comment by mertmit** [comment]
How did you setup your instance (docker-compose or docker run)? 
Attachment data should rely on the path you've mounted for `/usr/app/data/`
Database data depends on your NC_DB type (if you didn't provide you will see noco.db in same path)
I see mysql reference on the image you've shared so you should look for which path you've mounted for `/var/lib/mysql`
